container-scan/action.yaml

```diff
@@ -0,0 +1,92 @@
+name: "Security Scan Composite Action"
+description: "GitHub Action for scanning container image for vulnerabilities using Lacework"
+
+inputs:
+  #  docker-config-file:
+  #    required: true
+  #    description: Path to the docker config file
+  dockerhub-user:
+    required: true
+    description: DockerHub username
+  dockerhub-password:
+    required: true
+    description: DockerHub password
+  github-token:
+    required: true
+    description: GitHub token
+  artifactory-username:
+    required: true
+    description: Artifactory username
+  artifactory-auth-token:
+    required: true
+    description: Artifactory auth token
+  lw-account-name:
+    required: true
+    description: Lacework account name
+  lw-access-token:
+    required: true
+    description: Lacework access token
+  image-name:
```

Comment:
In other action we assume that there is a `docker-config.json` file with the iamge name. If you want to use that here too https://github.com/open-turo/actions-jvm/blob/993178f87d51d3c8f972403d58deb8ae59980fff/build-docker/action.yaml#L4

Reply:
I am trying to use the docker-config.json file, not sure if this implementation is correct